### ISSUE TITLE
The navigation icon appears on the top app bar instead of the back arrow

### ISSUE THREAD

**Issue by vvzuev** [federicoiosue/Omni-Notes#798]
Frequency: Sometime

Envoronment: Samsung Galaxy A70 - Android 10

Actions Performed: 
1. Open app
2. Tap the  Floating Action Button and select "Text note"
3. Create an any tag in the area "Content"
4. Go back to the screen "Notes"
5. Tap on the note you created in step 3
6. Click on a tag in the area "Content"
7. Tap the action "OPEN" in the appeared dialog

Expected Results: The top app bar holds the back arrow

Actual Results: The back arrow in the top app bar has been replaced with the navigation icon

Attachments: 
![omni_bag_2](https://user-images.githubusercontent.com/51853424/99877888-60cccc80-2c12-11eb-8501-160ae6f091e8.gif)

**Comment by rayauf** [comment]
I would like to work on this issue If it's possible

**Comment by federicoiosue** [comment]
> I would like to work on this issue If it's possible

Of course, but pleae pay attention to work on a test for your fix too (look at README for more details).

**Comment by rayauf** [comment]
Great, I take care of it, and the test too
